src/useRestActions.ts

```diff
@@ -29,73 +29,75 @@ const useRestActions = <Schema>(conf?: Partial<Config>, initialData?: Schema[])
 
   // index
   const indexConfig = mergeConfig(config.index, config)
-  const handleSuccessIndex = (result: Schema[]) => {
+  const handleSuccessIndex = useCallback((result: Schema[]) => {
     const { createSetItems } = actionCreators
     dispatch(createSetItems(result))
-  }
+  }, [ actionCreators, dispatch ])
   const index = useCreateAction<Schema>(handleSuccessIndex as HandleSuccess<Schema>, shouldMakeRequest(indexConfig) ? "GET" : undefined, indexConfig, actionCreators, dispatch)
-  const indexStable = useCallback(index, [])
+  const indexStable = useCallback(index, [ handleSuccessIndex, indexConfig, actionCreators, dispatch ])
 
   // create
   const createConfig = mergeConfig(config.create, config)
-  const handleSuccessCreate = (result: Schema) => {
+  const handleSuccessCreate = useCallback((result: Schema) => {
     const { createAddItem } = actionCreators
     dispatch(createAddItem(result))
-  }
+  }, [ actionCreators, dispatch ])
   const create = useCreateAction<Schema>(handleSuccessCreate as HandleSuccess<Schema>, shouldMakeRequest(createConfig) ? "POST" : undefined, createConfig, actionCreators, dispatch)
-  const createStable = useCallback(create, [])
+  const createStable = useCallback(create, [ handleSuccessCreate, createConfig, actionCreators, dispatch ])
 
   // read
   const readConfig = mergeConfig(config.read, config)
-  const handleSuccessRead = (result: Schema) => {
+  const handleSuccessRead = useCallback((result: Schema) => {
     const { createAddItem } = actionCreators
     dispatch(createAddItem(result))
-  }
+  }, [ actionCreators, dispatch ])
   const read = useCreateAction<Schema>(handleSuccessRead as HandleSuccess<Schema>, shouldMakeRequest(readConfig) ? "GET" : undefined, readConfig, actionCreators, dispatch)
-  const readStable = useCallback(read, [])
+  const readStable = useCallback(read, [ handleSuccessRead, readConfig, actionCreators, dispatch ])
 
   // update
   const updateConfig = mergeConfig(config.update, config)
-  const handleSuccessUpdate = (item: Schema) => {
+  const handleSuccessUpdate = useCallback((item: Schema) => {
     const { createUpdateItem, createAddItem } = actionCreators
     const index = identify(state.data, item, config.idKey)
     if (index > -1) {
       dispatch(createUpdateItem(index, item))
     } else {
       dispatch(createAddItem(item))
     }
-  }
+  }, [ actionCreators, state, config, dispatch ])
+
   const update = useCreateAction<Schema>(handleSuccessUpdate as HandleSuccess<Schema>, shouldMakeRequest(updateConfig) ? "PUT" : undefined, updateConfig, actionCreators, dispatch)
-  const updateStable = useCallback(update, [])
```

Comment:
Fooling the linter now becomes evident:
- `updateStable` is memoized but never renewed. (No dependencies).
- Its (among other things) dependent on: `handleSuccessUpdate` (but never renewed).
- `handleSuccessUpdate` is dependent on `state`, but as its never renewed it will take the state as it was given when initialized (state.data === undefined).